App/Models/TextileTypes.ts

```diff
@@ -3,6 +3,38 @@ export interface Event {
   payload: string
 }
 
+export enum NotificationType {
+  receivedInviteNotification,
+  deviceAddedNotification,
+  photoAddedNotification,
+  commentAddedNotification,
+  likeAddedNotification,
+  peerJoinedNotification,
+  peerLeftNotification
+}
+
+export interface Notification {
+  id: string
+  date: string
+  actor_id: string
+  actor_username: string
+  target_id: string
+  type: number
```

Comment:
Does this correspond to a `NotificationType`? If so, we should use the actual type here.